scripts/install-multi-user.sh

```diff
@@ -437,24 +437,17 @@ create_build_users() {
 }
 
 create_directories() {
```

Comment:
We probably need to think about how this effect current installs. Should Nix include some migration for them?

Reply:
No, `LocalStore::LocalStore()` automatically fixes permissions on `{profiles,gcroots}/per-user`.